{
  "comments": [
    {
      "key": {
        "uuid": "5a72e520_cbf3c008",
        "filename": "midolman/src/main/scala/org/midonet/containers/IPSecContainer.scala",
        "patchSetId": 8
      },
      "lineNbr": 396,
      "author": {
        "id": 1003318
      },
      "writtenOn": "2016-03-03T17:01:10Z",
      "side": 1,
      "message": "Is that possible? from what I see, if config is null on the catch, it means the container did not start in the first place and we failed to unsubscribe, or that we succesfully deleted the container (cleanup) but failed to unsubscribe. If you see this, it probably is the former which means something was odd at creation time.",
      "revId": "70c747f6d0ead28ed8891711bf3db4db708e2196",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "5a72e520_2b37f4e4",
        "filename": "midolman/src/main/scala/org/midonet/midolman/containers/ContainerService.scala",
        "patchSetId": 8
      },
      "lineNbr": 164,
      "author": {
        "id": 1003318
      },
      "writtenOn": "2016-03-03T16:54:44Z",
      "side": 1,
      "message": "remove.",
      "revId": "70c747f6d0ead28ed8891711bf3db4db708e2196",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    }
  ]
}